monai/transforms/spatial/array.py

```
@@ -1629,13 +1631,14 @@ def randomize(self, img: NdarrayOrTensor) -> None:
         super().randomize(None)
         if not self._do_transform:
             return None
+        _sp = get_spatial_ndim(img)
         self._zoom = [self.R.uniform(l, h) for l, h in zip(self.min_zoom, self.max_zoom)]
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)